README.md

```diff
@@ -139,9 +139,9 @@ Please run the quantization code first
 
 **cuda/cpu**: git clone https://github.com/intel/auto-round.git && cd auto-round && pip install -vvv --no-build-isolation -e .
 
-cpu: pip install intel-extension-for-transformers
+**cpu**: pip install intel-extension-for-transformers
```

Comment:
cuda:  git clone https://github.com/intel/auto-round.git && cd auto-round && pip install -vvv --no-build-isolation -e .
cpu: option 1 pip install auto-round && pip install intel-extension-for-transformers
        option2  git clone https://github.com/intel/auto-round.git && cd auto-round && pip install -vvv --no-build-isolation -e .